cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/run/GenericExtractionItemHandler.java

```diff
@@ -363,6 +364,12 @@ private void shutdown() {
         continue;
       }
       ExtractionItemContainer item = providerResult.get();
+
+      /* Skip directories. They will be traversed recursively anyway, but obviously the actual directory is not processed */
+      if (Files.isDirectory(item.getPathForExtraction())) {
+        continue;
```

Comment:
Have you checked if this still works with image sequences, which are stored as a directory of (flat) images?